release_notes/370-release-notes.adoc

```diff
@@ -0,0 +1,185 @@
+
+[id="scope-increase-rn"]
+=== Increased number of allowed inclusion and exclusion scopes
+Previously, {product-title-short} limited the number of allowed inclusion and exclusion scopes within a scope to ten each. This restriction has been removed.
+
+[id="notable-tech-changes-370"]
+== Notable technical changes
+
+Vulnerability scanning and reporting for {op-system-first} nodes has been disabled until scanning improvements are made for improved accuracy and to support full host-level scanning beyond just Kubernetes components. Currently, {op-system} uses National Vulnerability Database (NVD) vulnerability data for reporting vulnerabilities for Kubernetes components from {op-system}. In the enhanced version, vulnerability reporting will be based on Red Hat published security data. (ROX-10662)
+
+[id="deprecation-notices-370"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed, refer to the table below. Additional information on some removed or deprecated functionality is available after the table.
+
+In the table, features are marked with the following statuses:
+
+- GA: General Availability
+- TP: Technology Preview
+- DEP: Deprecated
+- REM: Removed
+
+.Deprecated and removed features tracker
+[cols="3,1,1,1",options="header"]
+|====
+|Feature |{product-title-short} 3.68 |{product-title-short} 3.69 | {product-title-short} 3.70
+
+|Ability to delete default policies
+|DEP
+|DEP
+|REM
+
+|Ability to add comments to alerts and processes
+|GA
+|DEP
+|DEP
+
+|Anchore, Tenable, and Docker Trusted registry integrations
+|GA
+|DEP
+|DEP
+
+|External authorization plug-in for scoped access control
+|GA
+|DEP
+|DEP
+
+|`FROM` option in the Disallowed Dockerfile line policy field
+|GA
+|GA
+|DEP
+
+|`RenamePolicyCategory` and `DeletePolicyCategory` API endpoints
+|GA
+|GA
+|DEP
+
+| `--rhacs` option for the `roxctl` helm output command
+| GA
+| DEP
+| DEP
+
+|Security policies without a `policyVersion`
+|DEP
+|DEP
+|REM
+
+|`/v1/policies` API endpoint response: *field* response body parameter.
+|DEP
+|DEP
+|REM
+
+|====
+
+[id="deprecated-features-370"]
+=== Deprecated features
+
+** *Anchore*, *Tenable*, and *Docker Trusted Registry* integrations: The {product-title-short} scanner supersedes these integrations.
```

Comment:
The other two bullets have everything before the `:` in bold. Should this match that?